integration-tests/bats/sql-jwt-auth.bats

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+load $BATS_TEST_DIRNAME/helper/query-server-common.bash
+load $BATS_TEST_DIRNAME/helper/gen_keys.bash
+
+make_repo() {
+  mkdir "$1"
+  cd "$1"
+  dolt init
+  cd ..
+}
+
+setup() {
+    skiponwindows "Missing dependencies"
+    setup_no_dolt_init
+    make_repo repo1
+}
+
+teardown() {
+    stop_sql_server
+    teardown_common
+}
+
+
+@test "sql-server: jwt auth from config" {
+    cd repo1
+    cp "$BATS_TEST_DIRNAME"/../../go/cmd/dolt/commands/sqlserver/testdata/chain_key.pem .
+    cp "$BATS_TEST_DIRNAME"/../../go/cmd/dolt/commands/sqlserver/testdata/chain_cert.pem .
+    let PORT="$$ % (65536-1024) + 1024"
+    TOKEN="`cat $BATS_TEST_DIRNAME/token.jwt`"
```

Comment:
Is this the right way to get the token from the jwt file? Seems like I might be doing something wrong in the python code below. When I run this test it hangs for a long time but if I generate a jwt and pass it in directly as the password it passes.

Also should I gitignore the jwks and jwt files? Seems like it would be annoying if there were changes every times someone runs bats

Reply:
Ignore this. Got test to pass and am removing jwt/jwks files at the end of the test now